newIDE/app/src/ExpressionAutocompletion/index.js

```diff
@@ -513,32 +521,118 @@ type InsertedAutocompletion = {|
   addDot?: ?boolean,
   addParameterSeparator?: ?boolean,
   addNamespaceSeparator?: ?boolean,
+  shouldConvertToString?: ?boolean,
 |};
 
 type ExpressionAndCaretLocation = {|
   expression: string,
   caretLocation: number,
 |};
 
+const findCompletedExpressionStartPosition = (expression: string) => {
```

Comment:
it took me a while to understand what we're doing here.
Maybe add a comment with an example of what this function does?